api/server/router/network/network_routes.go

```diff
@@ -25,6 +25,10 @@ func (n *networkRouter) getNetworksList(ctx context.Context, w http.ResponseWrit
 		return err
 	}
 
+	if err := netFilters.Validate(acceptedNetworkFilters); err != nil {
```

Comment:
`acceptedNetworkFilters` is now only used in this file, perhaps it should be moved here as well